templates/alloy-template/src/components/Standard.tsx

```diff
@@ -0,0 +1,86 @@
+import { contentType, Infer } from '@optimizely/cms-sdk';
+import { RichText } from '@optimizely/cms-sdk/react/richText';
+import {
+  ComponentContainerProps,
+  getPreviewUtils,
+  OptimizelyExperience,
+} from '@optimizely/cms-sdk/react/server';
+
+export const StandardContentType = contentType({
+  key: 'Standard',
```

Comment:
We seem to be a bit inconsistent with content type naming. Some content types have the suffix 'Page' and some not.